doc/Procedures.md

```diff
@@ -1,34 +1,3 @@
-# Procedures
-
-## Release check list
-- Make sure regression tests pass
-- Update release notes and README.md
-- Make sure the version number in `src/main/resources/application.conf` is correct. It is used
-when building the release.
-- Merge onto master branch, make sure [travis tests](https://travis-ci.org/dnanexus/dxWDL) pass
-
-- Build new externally visible release
-```
-./scripts/build_all_releases.sh --staging-token XXX --production-token YYY --docker-user UUU --docker-password WWW
-```
-this will take a while. It builds the release
-on staging, runs multi-region tests on staging, builds on production, and creates an easy to
-use docker image.
-
-- Update [releases](https://github.com/dnanexus-rnd/dxWDL/releases) github page, use the `Draft a new release` button, and upload a dxWDL.jar file.
-
-## Post release
-- Update the version number in `src/main/resources/application.conf`. We don't want
-to mix the experimental release, with the old code.
-
-
-## Coding guidelines
-
-- Unit tests should assert, and not print to the console
-- WDL test files belong in the top directory `test`
-- Code should comply with scalafmt default settings + what is in the
-  .scalafmt.conf file at the top level of the project
-
 ## Building
```

Comment:
Just remove this paragraph. We no longer use dxjava, and `build_jar.py` should not be used manually.

Reply:
Done. I moved the remaining Linux instruction sinto Developing.md.